snap_plugin/v1/integer_policy.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+
+# http://www.apache.org/licenses/LICENSE-2.0.txt
+#
+# Copyright 2016 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .plugin_pb2 import IntegerPolicy as PbIntegerPolicy
+from .plugin_pb2 import IntegerRule as PbIntegerRule
+
+
+class _IntegerPolicy(object):
```

Comment:
It's a private class that isn't exposed to the user and doesn't appear in the api docs.  The rules are documented and is the part of the policy that plugin authors will be exposed to.  Creating rules of a given type will automatically wrap them in the appropriate policy.  This all being said I'm going to add a brief doc string to all of the policies for those who might support the lib in the future.